app/src/main/java/org/ramonaza/officialramonapp/people/rides/backend/RidesOptimizer.java

```diff
@@ -13,21 +14,27 @@
 public class RidesOptimizer {
 
     /**
+     * Calculate rides based on latitude and longitude, finding the optimal assignment of passengers
+     * to drivers in order to minimize total distance traveled by all drivers. Makes the simplifying
+     * assumption that all drivers return to their home in between each drop-off.
+     */
+    public static final int ALGORITHM_NAIVE_HUNGARIAN=0;
```

Comment:
If possible I would prefer both picklist and code to be in order that the algorithms were added. Thank you.